code/stylesheets/spec.pcss

```diff
@@ -57,6 +57,7 @@
 
     pre {
       code {
+        display: block; /* make things smooth with js disabled */
```

Comment:
You can incorporate this right into the Tailwind `@apply` directive by just adding `block`:

```
@apply block my-8 p-4 whitespace-pre;
```